app/src/main/res/layout/feeditemlist_item.xml

```diff
@@ -189,14 +189,14 @@
                     style="@style/AntennaPod.TextView.ListItemSecondaryTitle"
                     tools:text="00:42:23" />
 
-                <ProgressBar
+                <com.google.android.material.progressindicator.LinearProgressIndicator
                     android:id="@+id/progressBar"
                     android:layout_width="0dp"
-                    android:layout_height="4dp"
+                    android:layout_height="wrap_content"
                     android:layout_weight="1"
                     android:max="100"
                     android:layout_margin="4dp"
-                    style="?android:attr/progressBarStyleHorizontal" />
```

Comment:
Is this style now unused?

Reply:
Yeah that was needed to tell the native progressbar whether or not it should be linear or circular. But with the new material design there are now to different components.

Reply:
Oh sorry, I was actually thinking of `progressBarTheme` in `styles.xml`. Didn't read that line properly. How about that theme? Is that still needed?

Reply:
Yeah, I'm not quite sure here. I looked into it set's the colors but from what I can tell it's the default colors anyway because the only component it's applied to  doesn't seem to have other colors than the other progressbars. And in any way the new indicators use slightly different colors, so I don't think that's needed any longer.